components/locid_transform/src/fallback/mod.rs

```diff
@@ -84,8 +58,34 @@ mod algorithms;
 /// assert_eq!(fallback_iterator.get(), &locale!("und").into());
 /// ```
 ///
+/// Unicode extension keywords take part in fallback, but [auxiliary keys]
```

Comment:
I'll merge this PR because I think we agree that it is a step in the right direction given the current state of how aux keys work. Users can click the link and read that aux keys are experimental. I think we'll need to change the machinery in 2.0.

Reply:
We decided to not merge this in https://github.com/unicode-org/icu4x/pull/4412#issuecomment-1852476099.